conf/logback.xml

```diff
@@ -14,12 +14,6 @@
         </encoder>
     </appender>
 
-    <appender name="STDOUT_IGNORE_NETTY" class="ch.qos.logback.core.ConsoleAppender">
```

Comment:
There seems to still be a reference to this appender on line 37, so doesn't that need removing too?

Reply:
Good spot, I'll remove that

Reply:
https://github.com/hmrc/register-trust-beneficiary-frontend/pull/249/commits/3c831fe9d19571411d620761af861fc2338da66c